### PR TITLE
fix/Removed object listing

### PR DESCRIPTION
**Pull request by carpawell**

### CODE REVIEW

**Review by roman-khimov — APPROVED** [review]

File: pkg/local_object_storage/metabase/select.go [hunk 1]
```diff
@@ -121,7 +121,7 @@ func (db *DB) selectObjects(tx *bbolt.Tx, cnr cid.ID, fs object.SearchFilters, c
 		addr.SetContainer(cnr)
 		addr.SetObject(id)
 
-		if objectStatus(tx, addr, currEpoch) > 0 {
+		if objectStatus(tx, addr, currEpoch) > 0 && !objectLocked(tx, cnr, id) {
```
Comment (cthulhu-rider) [inline pkg/local_object_storage/metabase/select.go:124]: did you think about return specific status from `objectStatus` for locked objects?

Comment (carpawell) [reply pkg/local_object_storage/metabase/select.go:124]: what is your idea exactly? in search operations be ready to return objects _and_ status code if _any_ locked objects have been found?

Comment (carpawell) [reply pkg/local_object_storage/metabase/select.go:124]: oh, sorry, got your wrong

Comment (carpawell) [reply pkg/local_object_storage/metabase/select.go]: found your idea usefull, check, please

File: CHANGELOG.md [hunk 2]
```diff
@@ -17,7 +17,8 @@ minor release, the component will be purged, so be prepared (see `Updating` sect
 - New `blobovnicza-to-peapod` tool providing blobovnicza-to-peapod data migration (#2453)
 - SN's version and capacity is announced via the attributes automatically but can be overwritten explicitly (#2455, #602)
 - `peapod` command for `neofs-lens` (#2507)
--  New CLI exit code for awaiting timeout (#2380)
+- New CLI exit code for awaiting timeout (#2380)
```
Comment (roman-khimov) [inline CHANGELOG.md:20]: Whitespace

File: pkg/local_object_storage/metabase/exists.go [hunk 3]
```diff
@@ -106,29 +106,41 @@ func objectStatus(tx *bbolt.Tx, addr oid.Address, currEpoch uint64) uint8 {
...
 			}
 		}
 	}
 
 	if expired {
+		if objectLocked(tx, cID, oID) {
+			return 0
+		}
+
 		return 3
```
Comment (roman-khimov) [inline pkg/local_object_storage/metabase/exists.go:131]: Love this.

**Review by roman-khimov — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (carpawell) [reply CHANGELOG.md:20]: on purpose. a sep commit? made it cause i added almost the same line but it looks strange cause a double space was added before

File: pkg/local_object_storage/metabase/exists.go (hunk 3 above)
Comment (carpawell) [reply pkg/local_object_storage/metabase/exists.go:131]: you mean magic num? add a const for it?

File: CHANGELOG.md (hunk 2 above)
Comment (roman-khimov) [reply CHANGELOG.md:20]: > a sep commit? 

Yep.

Comment (carpawell) [reply CHANGELOG.md:20]: fixed

**Pull request merged**